src/Directory.Build.props

```diff
@@ -71,6 +71,22 @@
     <DefineConstants>$(DefineConstants);ENCLAVE_SIMULATOR</DefineConstants>
   </PropertyGroup>
 
+  <!-- Packaging -->
+  <PropertyGroup>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!-- Repository Url is required for none SDK project even if it is already in nuspec file. -->
+    <RepositoryUrl>https://github.com/dotnet/sqlclient</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>      
+    <EmbedUnTrackedSources>true</EmbedUnTrackedSources>
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
This should not be enabled during local dev or the debugger won’t be able to find the local source files. You can add a condition. See: [https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link/#deterministic-builds](https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link/#deterministic-builds)

Reply:
Lines 83,84 and 86 were added to another PackageGroup with condition TF_BUILD = true for Azure CI.  Line 85 is required to build the nupkg with valid symbols.